pvlib/pvsystem.py

```diff
@@ -2920,11 +2944,34 @@ def pvwatts_dc(effective_irradiance, temp_cell, pdc0, gamma_pdc, temp_ref=25.):
     .. [1] A. P. Dobos, "PVWatts Version 5 Manual"
            http://pvwatts.nrel.gov/downloads/pvwattsv5.pdf
            (2014).
+    .. [2] B. Marion, "Comparison of Predictive Models for
+           Photovoltaic Module Performance,"
+           https://doi.org/10.1109/PVSC.2008.4922586,
```

Comment:
let's use the doi sphinx role
```suggestion
           :doi:`10.1109/PVSC.2008.4922586`,
```